tacs/mphys/coupling.py

```diff
@@ -25,7 +25,7 @@ def initialize(self):
 
     def setup(self):
         self.fea_assembler = self.options["fea_assembler"]
-        self.check_partials = self.options["check_partials"]
+        self.under_check_partials = self.options["check_partials"]
```

Comment:
For consistency, I think we should change the name of the option to `under_check_partials` too

Reply:
Since that would be backwards incompatible I'd prefer to avoid this. But if you feel strongly I can reconsider